src/qt/clientmodel.cpp

```diff
@@ -184,6 +189,11 @@ PeerTableModel *ClientModel::getPeerTableModel()
     return peerTableModel;
 }
 
+PeerTableSortProxy* ClientModel::peerTableSortProxy()
```

Comment:
778a64af209e4fa692a3aca8376ba1bd5e1af881

I don't think this belongs here, should be a member of `RPCConsole` - usually, sort/filter models are instanced for each view unless you want to share the same sorting/filter result in multiple views (I know only it's only used in `RPCConsole`).

Reply:
From the `##bitcoin-core-gui` IRC channel:
> \<promag> https://github.com/bitcoin-core/gui/pull/18 seems ready
> \<promag> https://github.com/bitcoin-core/gui/pull/18#discussion_r616065705 can be addressed later